code/modules/mob/living/carbon/carbon.dm

```diff
@@ -402,17 +402,17 @@
 		src << browse(null, t1)
 
 	if(href_list["hands"])
-		if(usr.incapacitated() || !Adjacent(usr)|| isanimal(usr))
+		if(usr.incapacitated() || !Adjacent(usr)|| (isanimal(usr) && !isgrinch(usr)))
```

Comment:
the amount of staggeringly ugly exceptions we're making for grinch here begs the question: is there ANY reason we should not just upgrade the grinch to at least a carbon that simply has a gremlin act spell? this way he would have REAL HANDS and not need constant exceptions for being an animal

Reply:
I asked that exact question on Discord when I was working on it and I agree with that sentiment, but I also really spent more efforts on that PR than I thought I would (how hard can it be to fix the Grinch I cluelessly told myself) and I'd like not to turn it into a carbon in this PR because I can forsee another all nighter to test and fix all the edge cases. If it's necessary to have the PR get merged then I'll let it sleep until I find the motivation to tackle the change properly.